# Running a pre-trained YOLO classification model on only VEL

In this notebook, we're trying to run the YOLO classification model that we saw on only the radial velocity (rotational speed) radar imagery, just as a test to see if single images might do better than trying to stuff all of the images from the radar imagery into one big image.

In [1]:
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n-cls.pt")  # load a pretrained model (recommended for training)

dataset_path = "/mnt/d/tornetExamples/vel_2014_and_2015"

# Train the model
results = model.train(data="/mnt/d/tornetExamples/vel_2014_and_2015", augment=False, epochs=10, save=False, batch=4, freeze=7, imgsz=240)

New https://pypi.org/project/ultralytics/8.3.39 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/vel_2014_and_2015, epochs=10, time=None, patience=100, batch=4, imgsz=240, save=False, save_period=-1, cache=False, device=None, workers=8, project=None, name=train27, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed

E0000 00:00:1732906793.486899    6468 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1732906793.630077    6468 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered


Overriding model.yaml nc=80 with nc=2

                   from  n    params  module                                       arguments                     
  0                  -1  1       464  ultralytics.nn.modules.conv.Conv             [3, 16, 3, 2]                 
  1                  -1  1      4672  ultralytics.nn.modules.conv.Conv             [16, 32, 3, 2]                
  2                  -1  1      6640  ultralytics.nn.modules.block.C3k2            [32, 64, 1, False, 0.25]      
  3                  -1  1     36992  ultralytics.nn.modules.conv.Conv             [64, 64, 3, 2]                
  4                  -1  1     26080  ultralytics.nn.modules.block.C3k2            [64, 128, 1, False, 0.25]     
  5                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
  6                  -1  1     87040  ultralytics.nn.modules.block.C3k2            [128, 128, 1, True]           
  7                  -1  1    295424  ultralytics

train: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/train... 37168 images, 0 corrupt: 100%|██████████| 37168/37168 [


train: New cache created: /mnt/d/tornetExamples/vel_2014_and_2015/train.cache


val: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/test... 6448 images, 0 corrupt: 100%|██████████| 6448/6448 [00:16<


val: New cache created: /mnt/d/tornetExamples/vel_2014_and_2015/test.cache
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000714, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 256 train, 256 val
Using 8 dataloader workers
Logging results to runs/classify/train27
Starting training for 10 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 76.88it/s]

                   all      0.896          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 77.74it/s]

                   all      0.901          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 75.12it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 76.46it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 75.41it/s]

                   all        0.9          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:10<00:00, 76.74it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:11<00:00, 70.04it/s]

                   all      0.899          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:11<00:00, 72.26it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:11<00:00, 71.96it/s]

                   all      0.902          1



      Epoch    GPU_mem       loss  Instances       Size


      10/10     0.134G     0.1887          4        256: 100%|██████████| 9292/9292 [02:24<00:00, 64.25it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 806/806 [00:11<00:00, 70.76it/s]

                   all      0.902          1



10 epochs completed in 0.450 hours.
Optimizer stripped from runs/classify/train27/weights/last.pt, 3.2MB


In [3]:
from ultralytics import YOLO

# Load a model
model = YOLO("yolo11n-cls.pt")  # load a pretrained model (recommended for training)

dataset_path = "/mnt/d/tornetExamples/vel_2014_and_2015"

# Train the model
results = model.train(data="/mnt/d/tornetExamples/vel_2014_and_2015", augment=False, epochs=20, save=False, batch=16, freeze=7, imgsz=240)

New https://pypi.org/project/ultralytics/8.3.39 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/vel_2014_and_2015, epochs=20, time=None, patience=100, batch=16, imgsz=240, save=False, save_period=-1, cache=False, device=None, workers=8, project=None, name=train29, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=7, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embe

train: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/train... 37168 images, 0 corrupt: 100%|██████████| 37168/37168 [
val: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/test... 6448 images, 0 corrupt: 100%|██████████| 6448/6448 [00:00<


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: SGD(lr=0.01, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 256 train, 256 val
Using 8 dataloader workers
Logging results to runs/classify/train29
Starting training for 20 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.28it/s]

                   all      0.895          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.54it/s]

                   all      0.902          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [01:06<00:00,  3.03it/s]

                   all      0.896          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.09it/s]

                   all        0.9          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.52it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.55it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.49it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.35it/s]

                   all      0.901          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.78it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


      10/20     0.191G     0.2025         16        256: 100%|██████████| 2323/2323 [01:12<00:00, 32.01it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.06it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


      11/20     0.224G     0.1998         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.64it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.16it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


      12/20     0.203G     0.1986         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 32.74it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.79it/s]

                   all      0.909          1



      Epoch    GPU_mem       loss  Instances       Size


      13/20     0.203G     0.1924         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 32.95it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.28it/s]

                   all      0.911          1



      Epoch    GPU_mem       loss  Instances       Size


      14/20     0.203G      0.193         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.68it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.17it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


      15/20     0.203G     0.1873         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 32.93it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.68it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


      16/20     0.203G     0.1835         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.58it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.68it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


      17/20     0.203G     0.1822         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.65it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.04it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


      18/20     0.203G     0.1783         16        256: 100%|██████████| 2323/2323 [01:15<00:00, 30.66it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.37it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


      19/20     0.203G     0.1741         16        256: 100%|██████████| 2323/2323 [01:18<00:00, 29.51it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:11<00:00, 17.28it/s]

                   all      0.902          1



      Epoch    GPU_mem       loss  Instances       Size


      20/20     0.203G      0.171         16        256: 100%|██████████| 2323/2323 [01:22<00:00, 28.09it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.09it/s]

                   all      0.903          1



20 epochs completed in 0.590 hours.
Optimizer stripped from runs/classify/train29/weights/last.pt, 3.2MB


In [4]:
results = model.train(data="/mnt/d/tornetExamples/vel_2014_and_2015", augment=False, epochs=20, save=False, batch=16, freeze=6, imgsz=240)

New https://pypi.org/project/ultralytics/8.3.39 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/vel_2014_and_2015, epochs=20, time=None, patience=100, batch=16, imgsz=240, save=False, save_period=-1, cache=False, device=None, workers=8, project=None, name=train292, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=6, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, emb

train: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/train... 37168 images, 0 corrupt: 100%|██████████| 37168/37168 [
val: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/test... 6448 images, 0 corrupt: 100%|██████████| 6448/6448 [00:00<


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: SGD(lr=0.01, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 256 train, 256 val
Using 8 dataloader workers
Logging results to runs/classify/train292
Starting training for 20 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.63it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.49it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.23it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.54it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.77it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.86it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.76it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.91it/s]

                   all      0.908          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.72it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


      10/20     0.218G     0.1821         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.60it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 21.34it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


      11/20     0.216G     0.1794         16        256: 100%|██████████| 2323/2323 [01:12<00:00, 32.16it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.94it/s]

                   all      0.908          1



      Epoch    GPU_mem       loss  Instances       Size


      12/20     0.216G     0.1743         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.40it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.09it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


      13/20     0.216G     0.1698         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.46it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.93it/s]

                   all      0.908          1



      Epoch    GPU_mem       loss  Instances       Size


      14/20     0.216G     0.1701         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 32.72it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.82it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


      15/20     0.216G     0.1636         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 33.16it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.54it/s]

                   all      0.902          1



      Epoch    GPU_mem       loss  Instances       Size


      16/20     0.216G     0.1615         16        256: 100%|██████████| 2323/2323 [01:10<00:00, 32.88it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.76it/s]

                   all      0.896          1



      Epoch    GPU_mem       loss  Instances       Size


      17/20     0.216G     0.1573         16        256: 100%|██████████| 2323/2323 [01:11<00:00, 32.27it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.80it/s]

                   all      0.898          1



      Epoch    GPU_mem       loss  Instances       Size


      18/20     0.216G     0.1525         16        256: 100%|██████████| 2323/2323 [01:13<00:00, 31.79it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.02it/s]

                   all        0.9          1



      Epoch    GPU_mem       loss  Instances       Size


      19/20     0.216G     0.1489         16        256: 100%|██████████| 2323/2323 [01:25<00:00, 27.28it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.91it/s]

                   all      0.899          1



      Epoch    GPU_mem       loss  Instances       Size


      20/20     0.216G     0.1439         16        256: 100%|██████████| 2323/2323 [01:14<00:00, 30.99it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.89it/s]

                   all      0.899          1



20 epochs completed in 0.458 hours.
Optimizer stripped from runs/classify/train292/weights/last.pt, 3.2MB


In [5]:
results = model.train(data="/mnt/d/tornetExamples/vel_2014_and_2015", augment=False, epochs=20, save=False, batch=16, freeze=5, imgsz=240)

New https://pypi.org/project/ultralytics/8.3.39 available 😃 Update with 'pip install -U ultralytics'
Ultralytics 8.3.36 🚀 Python-3.9.20 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 3070, 8192MiB)
engine/trainer: task=classify, mode=train, model=yolo11n-cls.pt, data=/mnt/d/tornetExamples/vel_2014_and_2015, epochs=20, time=None, patience=100, batch=16, imgsz=240, save=False, save_period=-1, cache=False, device=None, workers=8, project=None, name=train2922, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=5, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, em

train: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/train... 37168 images, 0 corrupt: 100%|██████████| 37168/37168 [
val: Scanning /mnt/d/tornetExamples/vel_2014_and_2015/test... 6448 images, 0 corrupt: 100%|██████████| 6448/6448 [00:00<


optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: SGD(lr=0.01, momentum=0.9) with parameter groups 39 weight(decay=0.0), 40 weight(decay=0.0005), 40 bias(decay=0.0)
TensorBoard: model graph visualization added ✅
Image sizes 256 train, 256 val
Using 8 dataloader workers
Logging results to runs/classify/train2922
Starting training for 20 epochs...

      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:51<00:00,  3.95it/s]

                   all      0.897          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.27it/s]

                   all      0.897          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.70it/s]

                   all      0.897          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.83it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.11it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.20it/s]

                   all        0.9          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.08it/s]

                   all      0.911          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.83it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.62it/s]

                   all      0.905          1



      Epoch    GPU_mem       loss  Instances       Size


      10/20      0.22G     0.1694         16        256: 100%|██████████| 2323/2323 [01:16<00:00, 30.32it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:11<00:00, 18.16it/s]

                   all      0.899          1



      Epoch    GPU_mem       loss  Instances       Size


      11/20      0.22G     0.1676         16        256: 100%|██████████| 2323/2323 [01:19<00:00, 29.39it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 19.35it/s]

                   all      0.904          1



      Epoch    GPU_mem       loss  Instances       Size


      12/20      0.22G     0.1618         16        256: 100%|██████████| 2323/2323 [01:16<00:00, 30.36it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 20.14it/s]

                   all      0.907          1



      Epoch    GPU_mem       loss  Instances       Size


      13/20      0.22G     0.1559         16        256: 100%|██████████| 2323/2323 [01:16<00:00, 30.46it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:11<00:00, 18.16it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


      14/20      0.22G     0.1558         16        256: 100%|██████████| 2323/2323 [01:19<00:00, 29.24it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 18.54it/s]

                   all      0.906          1



      Epoch    GPU_mem       loss  Instances       Size


      15/20      0.22G     0.1485         16        256: 100%|██████████| 2323/2323 [01:17<00:00, 29.82it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 18.97it/s]

                   all      0.903          1



      Epoch    GPU_mem       loss  Instances       Size


      16/20      0.22G     0.1488         16        256: 100%|██████████| 2323/2323 [01:17<00:00, 29.89it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.77it/s]

                   all      0.896          1



      Epoch    GPU_mem       loss  Instances       Size


      17/20      0.22G      0.144         16        256: 100%|██████████| 2323/2323 [01:13<00:00, 31.61it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.61it/s]

                   all      0.899          1



      Epoch    GPU_mem       loss  Instances       Size


      18/20      0.22G     0.1373         16        256: 100%|██████████| 2323/2323 [01:17<00:00, 30.04it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:11<00:00, 16.95it/s]

                   all        0.9          1



      Epoch    GPU_mem       loss  Instances       Size


      19/20      0.22G     0.1353         16        256: 100%|██████████| 2323/2323 [01:21<00:00, 28.45it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:09<00:00, 20.39it/s]

                   all      0.896          1



      Epoch    GPU_mem       loss  Instances       Size


      20/20      0.22G     0.1303         16        256: 100%|██████████| 2323/2323 [01:19<00:00, 29.24it/s]
               classes   top1_acc   top5_acc: 100%|██████████| 202/202 [00:10<00:00, 18.87it/s]

                   all      0.895          1



20 epochs completed in 0.573 hours.
Optimizer stripped from runs/classify/train2922/weights/last.pt, 3.2MB


We got as high as .911, but even when freezing fewer layers, we're not seeing much improvement in accuracy unfortunately.